dnf5-plugins/config-manager_plugin/addrepo.cpp

```diff
@@ -346,6 +346,9 @@ void ConfigManagerAddRepoCommand::add_repos_from_repofile(
         // Test if the repository options can be set.
         for (const auto & [repo_id, repo_opts] : parser.get_data()) {
             for (const auto & [key, key_val] : repo_opts) {
+                if (key.starts_with("#")) {
```

Comment:
I understand this fixes the comment lines, but how it fixes the empty ones?

Reply:
Or at least at the comment here why it is skipping empty and comment lines, because just looking at this, it's a bit mysterious to me :)

Reply:
OK. I added a comment.

Reply:
Thanks!